src/js/bootstrap-slider.js

```diff
@@ -1261,6 +1269,16 @@
 
 				return true;
 			},
+			_touchstart: function(ev) {
+				if (ev.changedTouches === undefined) {
+					this._trigger('touchStart');
```

Comment:
this also needs to emit the current value of the slider (similar to this line): https://github.com/jerrylow/bootstrap-slider/blob/Touch-Fix/src/js/bootstrap-slider.js#L1326